content/_guides/creatingchannels.md

```diff
@@ -1,52 +1,211 @@
 ---
 title: Creating Channels
 category: chanop
-credits: web7
 ---
 
+A person who joins an _empty_ and unregistered channel is granted
+**temporary** Operator status. This does not guarantee their ownership of the
+channel. They need to `REGISTER` the channel with `ChanServ` to gain ownership
+of it.
+
+This guide will help you prepare for and go through that process.
+
 ## Choosing the correct namespace
 
 Before creating a new channel on Libera.Chat, please take a moment to read up
 on [our namespaces policy](/chanreg) and ensure that your channel is named
 appropriately from the beginning. This will avoid disruptions in the future.
 
-## How to create a new channel
+## Be identified
+
+To register and manage channels, you will need to be [identified to](registration#logging-in)
+`NickServ`.
+
+## Pick a name
+
+Before doing that, you'll obviously need to pick a channel name! Think of one
```

Comment:
Before doing what?

Reply:
It probably made sense before last night's rerewrite